docs/tools/vdb_table/data/vespa.json

```diff
@@ -73,33 +73,33 @@
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/vespa-engine/sample-apps/tree/master/text-image-search",
     "comment": ""
   },
   "embeddings_structured": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Through the support for custom ONNX models, Vespa also supports structured embeddings"
   },
   "rag": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.vespa.ai/en/llms-rag.html",
     "comment": ""
   },
   "recsys": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.vespa.ai/en/tutorials/news-4-embeddings.html",
```

Comment:
Similar story to my other comments here - this is not built-in recsys, this is "you can build recsys with vespa".

But Looking at the recsys column, we let this slip for other vendors so it's fair to put this as "full" and we will clean that up in one fell swoop later.

The linked tutorial is super useful!